be/src/exec/exec_node.cpp

```diff
@@ -163,6 +163,7 @@ Status ExecNode::init_join_runtime_filters(const TPlanNode& tnode, RuntimeState*
             register_runtime_filter_descriptor(state, rf_desc);
         }
     }
+    _runtime_filter_collector.set_plan_node_id(_id);
```

Comment:
why need to set plan node is here?

Reply:
we can set condtional breakpoints in gdb when we only focus on runtime filters on a certain plan node.